tests/common.go

```diff
@@ -3439,14 +3439,14 @@ func DeleteBackup(backupName string, backupUID string, orgID string, ctx context
 }
 
 // DeleteCluster deletes/de-registers cluster from px-backup
-func DeleteCluster(name string, orgID string, ctx context1.Context) error {
+func DeleteCluster(name string, orgID string, ctx context1.Context, cleanupBackupsRestores bool) error {
```

Comment:
Can we have two Boolean parameters for DeleteBackups and DeleteRestores?

Reply:
I thought about that. But this is mainly for cleanup. We will either cleanup everything or not cleanup at all.
This cleanup can only be done from the API, from API we can never cleanup the backups and restores while deleting the cluster.